pkg/controller/jenkins/configuration/base/reconcile.go

```diff
@@ -853,3 +854,14 @@ func (r *ReconcileJenkinsBaseConfiguration) ensureBaseConfiguration(jenkinsClien
 
 	return reconcile.Result{Requeue: requeue}, err
 }
+
+func (r *ReconcileJenkinsBaseConfiguration) isOpenshiftEnabled() bool {
+	openshiftEnabled := false
+	if r.Configuration.Jenkins.Annotations[openshiftEnabledAnnotation] == "true" {
+		openshiftEnabled = true
+		r.logger.Info("-Openshift Mode-")
+	} else {
+		openshiftEnabled = false
+	}
```

Comment:
remove this, this doesn't do anything. When removed, if the condition on line 860 won't be satisfied, then the `openshiftEnabled` will be `false`

Reply:
good catch on it